src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -109,9 +109,10 @@ private static void InitializeProcess()
             {
                 AppDomain.CurrentDomain.UnhandledException += CurrentDomainOnUnhandledExceptionInLinuxConsumption;
             }
-            else if (SystemEnvironment.Instance.IsFlexConsumptionSku())
+            else if (SystemEnvironment.Instance.IsFlexConsumptionSku() ||
```

Comment:
While making the other changes, I reviewed all the other places where we're checking for Flex SKU, and realized that we needed to make a change here as well.